crates/modelardb_server/src/data_folders.rs

```diff
@@ -0,0 +1,216 @@
+/* Copyright 2024 The ModelarDB Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//! Implementation of a struct that provides access to the local and remote data storage components.
+
+use std::sync::Arc;
+
+use deltalake_core::DeltaTableError;
+use modelardb_common::metadata::table_metadata_manager::TableMetadataManager;
+use modelardb_common::storage::DeltaLake;
+use modelardb_common::types::ServerMode;
+
+use crate::manager::Manager;
+use crate::ClusterMode;
+
+/// Folder for storing metadata and Apache Parquet files.
```

Comment:
Are both data and metadata not stored in Apache Parquet files? If so, maybe update the doc comment here and in the rest of the file.

Reply:
Fixed.